src/Compilers/Server/VBCSCompiler/AnalyzerConsistencyChecker.cs

```diff
@@ -17,7 +17,12 @@ internal static class AnalyzerConsistencyChecker
     {
         private static readonly ImmutableArray<string> s_defaultIgnorableReferenceNames = ImmutableArray.Create("mscorlib", "System", "Microsoft.CodeAnalysis", "netstandard");
 
-        public static bool Check(string baseDirectory, IEnumerable<CommandLineAnalyzerReference> analyzerReferences, IAnalyzerAssemblyLoader loader, IEnumerable<string>? ignorableReferenceNames = null)
+        public static bool Check(
+            string baseDirectory,
+            IEnumerable<CommandLineAnalyzerReference> analyzerReferences,
+            IAnalyzerAssemblyLoader loader,
+            IEnumerable<string>? ignorableReferenceNames = null,
+            ICompilerServerLogger? logger = null)
```

Comment:
When would we actually want to not pass a logger?

Reply:
Good point. Removed the annotation here and fixed up the callers.